mobile/src/Student/screens/Game/BasicGamePlay/index.js

```diff
@@ -14,22 +21,31 @@ import HintsView from "../Components/HintsView"
 import { GameSessionState } from "@righton/networking"
 import uuid from "react-native-uuid"
 
-const GamePreview = ({ navigation, route, gameSession, team, teamMember }) => {
-    const question = gameSession.isAdvanced
-        ? team.question
-        : gameSession.questions[
-              gameSession.currentQuestionIndex == null
-                  ? 0
-                  : gameSession.currentQuestionIndex
-          ]
-    const availableHints = question.instructions
+const BasicGamePlay = ({
+    navigation,
+    route,
+    gameSession,
+    teamId,
+    teamMember,
+}) => {
+    const team = gameSession?.teams.find((team) => team.id === teamId)
+    console.log("team in BasicGamePlay", team)
+    // const question = gameSession?.isAdvanced
+    //     ? team.question
+    //     : gameSession?.questions[
+    //           gameSession?.currentQuestionIndex == null
+    //               ? 0
+    //               : gameSession?.currentQuestionIndex
+    //       ]
+    const question = gameSession?.questions[1]
+    const availableHints = JSON.parse(question.instructions)
```

Comment:
`instructions` should've already been parsed here, if not, this code needs to be done in `networking` or backend if backend is sending double stringified json string